util/config.go

```diff
@@ -209,6 +218,20 @@ func validateConfig() {
 	}
 }
 
+func validateExternalAuth() {
```

Comment:
I just thought of it now, but I think it should be "external auth" or LDAP or internal. Not on top of it.

So, on that note, it should probably "error" when LDAP is enabled as well. Haven't found a simple way to determine "internal auth" yet.